storage/storage.go

```diff
@@ -28,3 +30,9 @@ type Market interface {
 	SaveRates(rates watchmarket.Rates, pl ProviderList) map[SaveResult]int
 	GetRate(currency string) (*watchmarket.Rate, error)
 }
+
+type Middleware interface {
```

Comment:
nit: i think `Middleware` for this interface is misleading.

Reply:
Can you suggest something more suitable, please?